### PR TITLE
Health warden

### PR DESCRIPTION
**Pull request by misslecter**

### CODE REVIEW

File: projects/health-warden.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  "desc_de": "Heart Rate Detector ist eine mobile App mit der Möglichkeit, den von einer Smartphone-Kamera erfassten Herzpuls zu messen, auszuwerten und zu teilen. Die Anwendung ermöglicht sowohl dem Patienten als auch dem Arzt, aufgezeichnete Daten über die Zeit zu sammeln und mit ihnen sowohl maßstabsgetreu für den behandelnden Arzt als auch in anonymisierter Form massenhaft zu arbeiten. Weitere Funktionen sind der Export von Daten im CSV-Format, das Anzeigen der Patientengeschichte oder das Senden von Einladungscodes, um Patienten mit einem Arzt zu verbinden. Die Anwendung steht den Nutzern kostenlos zur Verfügung und ermöglicht eine Verbesserung und Überwachung der Gesundheitsversorgung ohne zusätzliche Kosten für einen Patienten oder ein Krankenhaus.",
+  "contact_note_cs": "Máte zájem o využití aplikace HealthWarden ve Vaší nemocnice či ordinace? Kontaktujte nás na e-mailu níže a zajistíme Vám zdarma testovácí prostředí připravené na míru.",
```
Comment (notsousual) [inline projects/health-warden.json:13]: ordinaci

Comment (notsousual) [reply projects/health-warden.json:13]: Kontaktujte nás e-mailem

File: projects/health-warden.json [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  "contact_note_en": "Would you like to use HealthWarden app in your hospital or general practice? Contact us by e-mail below a to get customized test environment for free.",
+  "desc_pieces_cs": [
+    "HealthWarden je mobilní a desktopová aplikace pro komplexní ambulantní péči. Průměrné náklady na hospitalizaci za den přesahují 55 000 Kč, což zhoršuje dostupnost zdravotní péče. Přeplněnost nemocnic blokuje vykonání jiných plánovaných i akutních zákroků. Lékaři každý den stojí před těžkým rozhodnutím: nechat pacienta v nemocnici, aby bylo vše v pořádku, nebo poslat pacienta domů, aby se uvolnila kapacita nemocnice?",
```
Comment (notsousual) [inline projects/health-warden.json:16]: aby vse bylo v poradku
carka pred nebo asi neni potreba

za ucelem uvolneni kapacity nemocnice?

File: projects/health-warden.json [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    "Aplikace HealthWarden zkracuje dobu hospitalizace a poskytuje hladkou a bezpečnou komunikaci mezi lékařem a pacientem během ambulantní léčby. Aplikace provádí pravidelné kontroly zdravotního stavu pacienta pomocí prediktivních AI algoritmů. Jakékoli potenciální problémy jsou hlášeny lékaři pomocí nástrojů pro detekci anomálií. Každý pacient takto získává strukturovanou personalizovanou péči založenou na individuálních požadavcích, zatímco lékařům je poskytnutá dobře strukturovaná a snadno dostupná data o každém pacientovi i jeho stavu."
```
Comment (notsousual) [inline projects/health-warden.json:17]: Jakékoliv je asi oficialnejsi? nejsem si jista

File: projects/health-warden.json [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  "desc_en": "Heart Rate Detector is a mobile app with the possibility to measure, evaluate and share heart pulse detected by a smartphone camera. The application allows both patient and doctor to collect recorded data over time and work with them both on a scale for the doctor taking care of the patient and in an anonymized form on a mass scale. Other functionalities are exporting data in CSV format, viewing patients' history, or sending invitation codes to connect patients with a doctor. The application is available free of charge for users, enabling improvement and monitoring of health care without any additional cost for a patient or a hospital.",
```
Comment (notsousual) [inline projects/health-warden.json]: Other functionalities are data export in CSV format, viewing patient history or sending invitation codes connecting patients and doctors. 

carka pred or je spatne

**Review by notsousual — CHANGES_REQUESTED** [review]

File: projects/health-warden.json [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+      "icon": "ai.svg",
+      "title_cs": "Maximální automatizace díky AI",
+      "title_en": "Maximal automation thanks to robust AI powered algorithms",
+      "description_cs": "Aplikace automaticky provádí vyhodnocení stavu pacienta a upozorní lékaře až v případě komplikací.",
+      "description_en": "The app automatically evaluates the patient's condition and notifies the doctor only in case of complications."
+    },
+    {
+      "icon": "platform.svg",
+      "title_cs": "Podpora vícero platforem",
+      "title_en": "Multiplatform support",
```
Comment (notsousual) [inline projects/health-warden.json:52]: staci jedno slovo:
 Cross-Platform
 
je vicero spisovne?

File: projects/health-warden.json [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+      "description_cs": "Na webu, v mobilním telefonu a tabletu s Android či iOS. Vždy po ruce!",
+      "description_en": "On the web, in your Android or iOS phone and tablet. Always at your fingertips!"
```
Comment (notsousual) [inline projects/health-warden.json:54]: Available on the web, any Android or iOS device: always at your fingertips!

File: projects/heart.json [hunk 7]
```diff
@@ -12,7 +12,7 @@
   "desc_cs": "Heart Rate Detector je mobilní aplikace umožňující měřit, vyhodnocovat a sdílet srdeční tep detekovaný kamerou chytrého telefonu. Aplikace umožňuje pacientovi i lékaři sbírat zaznamenaná data a pracovat s nimi s cílem prevence či léčby srdečních onemocnění. Dalšími funkcemi jsou export dat ve formátu CSV, prohlížení historie pacientů nebo zasílání zvacích kódů pro spojení pacientů s lékařem. Aplikace je pro uživatele k dispozici zdarma a umožňuje zkvalitňování a sledování zdravotní péče bez dalších nákladů pro pacienta nebo nemocnici.",
```
Comment (notsousual) [inline projects/heart.json:12]: tep je vzdy srdecni,srdecni tep je castecne tautologie

File: projects/heart.json [hunk 8]
```diff
@@ -12,7 +12,7 @@
...
-  "contact_note_cs": "Máte zájem o využití aplikace Heart Rate Detector ve Vaší nemocnice či ordinace? Kontaktujte nás na e-mailu níže a zajistíme Vám zdarma testovácí prostředí připravené na míru.",
+  "contact_note_cs": "Máte zájem o využití aplikace Heart Rate Detector ve Vaší nemocnici či ordinaci? Kontaktujte nás e-mailem a zajistíme Vám zdarma testovácí prostředí připravené na míru.",
   "contact_note_en": "Would you like to use Heart Rate Detector app in your hospital or general practice? Contact us by e-mail below a to get customized test environment for free.",
   "desc_pieces_cs": [
```
Comment (notsousual) [inline projects/heart.json:17]: srdecni tep zase

File: projects/heart.json [hunk 9]
```diff
@@ -12,7 +12,7 @@
...
-  "contact_note_cs": "Máte zájem o využití aplikace Heart Rate Detector ve Vaší nemocnice či ordinace? Kontaktujte nás na e-mailu níže a zajistíme Vám zdarma testovácí prostředí připravené na míru.",
+  "contact_note_cs": "Máte zájem o využití aplikace Heart Rate Detector ve Vaší nemocnici či ordinaci? Kontaktujte nás e-mailem a zajistíme Vám zdarma testovácí prostředí připravené na míru.",
   "contact_note_en": "Would you like to use Heart Rate Detector app in your hospital or general practice? Contact us by e-mail below a to get customized test environment for free.",
```
Comment (notsousual) [inline projects/heart.json:16]: Contact us by e-mail below **a** to get
'a' zbytecne, asi preklep

File: projects/heart.json [hunk 10]
```diff
@@ -12,7 +12,7 @@
...
   "desc_en": "Heart Rate Detector is a mobile app with the possibility to measure, evaluate and share heart pulse detected by a smartphone camera. The application allows both patient and doctor to collect recorded data over time and work with them both on a scale for the doctor taking care of the patient and in an anonymized form on a mass scale. Other functionalities are exporting data in CSV format, viewing patients' history, or sending invitation codes to connect patients with a doctor. The application is available free of charge for users, enabling improvement and monitoring of health care without any additional cost for a patient or a hospital.",
```
Comment (notsousual) [inline projects/heart.json:13]: Other functionalities are data export in CSV format, viewing patient history, and sending invitation codes connecting patients and doctors.

Comment (notsousual) [reply projects/heart.json:13]: without any additional costs for patients and hospitals

**Review by notsousual — CHANGES_REQUESTED** [review]

File: projects/health-warden.json (hunk 4 above)
Comment (notsousual) [reply projects/health-warden.json]: actually grammarly pise ze je tam spravne, ale or predpoklada antitezi takze
Other functionalities are data export in CSV format, viewing patient history, and sending invitation codes connecting patients and doctors.

File: projects/health-warden.json [hunk 11]
```diff
@@ -0,0 +1,57 @@
...
+    {
+      "icon": "doctor.svg",
+      "title_cs": "Sdílení výsledků s lékařem",
+      "title_en": "Results sharing with a doctor",
+      "description_cs": "Výsledky měření jsou automaticky sdíleny s lékařem.",
+      "description_en": "The measurement results are automatically shared with a doctor."
+    },
+    {
+      "icon": "ai.svg",
+      "title_cs": "Maximální automatizace díky AI",
```
[the remark this answers is not in the document]
Comment (notsousual) [reply projects/health-warden.json:44]: ok

File: projects/health-warden.json [hunk 12]
```diff
@@ -0,0 +1,57 @@
...
+  "contact_note_cs": "Máte zájem o využití aplikace HealthWarden ve Vaší nemocnici? Kontaktujte nás e-mailem a přípravíme Vám nabídku na míru.",
+  "contact_note_en": "Would you like to use HealthWarden app in your hospital? Contact us by e-mail below a to get a customized offer.",
+  "desc_pieces_cs": [
+    "HealthWarden je mobilní a desktopová aplikace pro komplexní ambulantní péči. Průměrné náklady na hospitalizaci za den přesahují 55 000 Kč, což zhoršuje dostupnost zdravotní péče. Přeplněnost nemocnic blokuje vykonání jiných plánovaných i akutních zákroků. Lékaři každý den stojí před těžkým rozhodnutím: nechat pacienta v nemocnici aby zůstal pod dohledem odborníků, anebo poslat pacienta domů za účelem uvolnění kapacity nemocnice?",
```
[the remark this answers is not in the document]
Comment (misslecter) [reply projects/health-warden.json:16]: podle vseho ano, musi pred aby byt carka

**Review by notsousual — APPROVED** [review]

**Pull request merged**